src/context/schemas/action.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/action.schema.json",
+  "type": "object",
+  "title": "Action",
+  "properties": {
```

Comment:
missing `"allOf": [{ "$ref": "context.schema.json#" }],` above `porperties` to import the context schema and extend it

Reply:
See other context examples, e.g.: https://fdc3.finos.org/schemas/2.0/contact.schema.json

Reply:
```suggestion
  "allOf": [{ "$ref": "context.schema.json#" }],
  "properties": {
```